skops/saving_utils/model_card.py

```diff
@@ -0,0 +1,55 @@
+from modelcards import CardData, ModelCard
+from sklearn import set_config
+from sklearn.utils import estimator_html_repr
+
+
+def _plot_estimator(model):
+    set_config(display="diagram")
+    return estimator_html_repr(model)
+
+
+def _extract_estimator_config(model):
```

Comment:
this should be tested for cases when the huperparameters are non-trivial, like a custom scorer passed to a GridSearchCV

Reply:
just to confirm what I understood, Grid Search CV object itself returns best estimator no? How is it different? what do you mean with custom scorer? 😅

Reply:
`GridSeachCV` doesn't return the best estimator. It stores the best estimator in the `best_estimator_` attribute and uses it when you call `predict` on a `GridSearchCV` instance. For instance, you can see how the parameters of the `gs` here look like: https://scikit-learn.org/stable/auto_examples/model_selection/plot_multi_metric_evaluation.html#running-gridsearchcv-using-multiple-evaluation-metrics

Reply:
Ah maybe I remembered it like that because that's how I always use it 😅

Reply:
it still returns stuff we can parse, no? see below:

```
gs.get_params()
{'cv': None,
 'error_score': nan,
 'estimator': DecisionTreeClassifier(random_state=42),
 'estimator__ccp_alpha': 0.0,
 'estimator__class_weight': None,
 'estimator__criterion': 'gini',
 'estimator__max_depth': None,
 'estimator__max_features': None,
 'estimator__max_leaf_nodes': None,
 'estimator__min_impurity_decrease': 0.0,
 'estimator__min_samples_leaf': 1,
 'estimator__min_samples_split': 2,
 'estimator__min_weight_fraction_leaf': 0.0,
 'estimator__random_state': 42,
 'estimator__splitter': 'best',
 'n_jobs': 2,
 'param_grid': {'min_samples_split': range(2, 403, 20)},
 'pre_dispatch': '2*n_jobs',
 'refit': 'AUC',
 'return_train_score': True,
 'scoring': {'AUC': 'roc_auc', 'Accuracy': make_scorer(accuracy_score)},
 'verbose': 0}
```

Reply:
we can parse and persist them, but the `make_scorer(accuracy_score)` is code, it's not a literal or a number or the usual stuff you'd expect. We just need to make sure such a case is tested with the way you're creating the table.